kitsune/sumo/static/sumo/css/print.css

```diff
@@ -5,14 +5,52 @@ body {
     margin: 0;
     padding: 0;
 }
+
+body.scroll-header {
+    padding-top: 0;
+}
+
+body > header {
+    padding-bottom: 5px;
+    position: relative;
+}
+
+body > header a {
+    border-bottom: 0;
+}
+
+body.html-ltr .container_12 {
+    width: auto;
```

Comment:
Oops. This isn't about alphabetizing the selectors. This is about alphabetizing the rules in a selector. So this should be

```css
body.html-ltr .container_12 {
  margin: 0;
  padding: 0;
  width: auto;
}
```